deploy/charts/rook-ceph/templates/role.yaml

```diff
@@ -84,6 +84,9 @@ rules:
   - apiGroups: ["apps"]
     resources: ["deployments/finalizers", "daemonsets/finalizers"]
     verbs: ["update"]
+  - apiGroups: ["authentication.k8s.io"]
```

Comment:
we do not need for cephfs-nodeplugn?

Reply:
Not required for cephfsplugin as we dont run csi-addons with it